### PR TITLE
FPL-532 and 533 number bug fix

### PR DESCRIPTION
**Pull request by kurt-uk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPL-532
https://tools.hmcts.net/jira/browse/FPL-533

### Change description ###

Spoken to Chris regarding the split between Number and PhoneUK being used, had feedback that Number should be used instead so changed all instances to Number

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by kurt-uk** [comment]
@swalker125 this is the issue. The initial fix just replacing an oversight of a phone number field being a `Text` field type. we had `PhoneUK` in some places and `Number` in other places, `PhoneUK` isnt what we want becuase we dont want to restrict the numbers when international numbers could be needed. So its not a sense of its not working anymore, its a sense of its not right for what we need.

**Comment by kurt-uk** [comment]
@swalker125 this is the issue. The initial fix just replacing an oversight of a phone number field being a `Text` field type. we had `PhoneUK` in some places and `Number` in other places, `PhoneUK` isnt what we want becuase we dont want to restrict the numbers when international numbers could be needed. So its not a sense of its not working anymore, its a sense of its not right for what we need according to Chris

**Comment by kurt-uk** [comment]
asked CCD about the issue and they have said that PhoneUK isnt correctly named, and that it accepts non UK numbers with the use of `+`

**Comment by swalker125** [comment]
> asked CCD about the issue and they have said that PhoneUK isnt correctly named, and that it accepts non UK numbers with the use of `+`

Have you spoken to Olivia / Eilis if this is clear to users? Are you saying an international phone number will fail w

**Comment by swalker125** [comment]
Apologies - closed by accident.

**Comment by kurt-uk** [comment]
ive spoke to both and they were for the regex fix i included, however, since that time, CCD have gotten back to me and told me of their outdated name

**Comment by kurt-uk** [comment]
FOR CLARITY:
CCD informed me that their PhoneUK would accept international phone numbers, this isnt the case as we tried out with a Spanish phone number. informed ccd that this is the case, and reverted again to the Text field that applies a RegEx

**Comment by kurt-uk** [comment]
@swalker125 just to bring you up to speed:
CCD have gone back on their PhoneUK being able to accomodate international phone numbers, this isnt something they want to do at the moment, changed back to Text fields and applied additional regex that will allow numbers, +, and ()

**Comment by kurt-uk** [comment]
These formats are rejected:
screenshot 2019-01-17 at 16 38 08
screenshot 2019-01-17 at 16 38 31
These are acceptable formats:
screenshot 2019-01-17 at 16 38 49
screenshot 2019-01-17 at 16 39 08
screenshot 2019-01-17 at 16 39 16

**Comment by geek-soft** [comment]
@kurt-uk when you merge it, please make sure in commit message that this allows non UK phone numbers across app and addresses inconsistency issue we found.

File: ccd-definition/ComplexTypes.json [hunk 1]
```diff
@@ -196,6 +196,7 @@
     "ID": "Child",
     "ListElementCode": "socialWorkerTel",
```
Comment (geek-soft) [inline ccd-definition/ComplexTypes.json]: What about `socialWorkerPhone`?

File: ccd-definition/ComplexTypes.json [hunk 2]
```diff
@@ -323,17 +324,19 @@
     "LiveFrom": "01/01/2017",
     "ID": "Applicant",
     "ListElementCode": "mobile",
```
Comment (geek-soft) [inline ccd-definition/ComplexTypes.json]: What about `mobilePhone`?

File: ccd-definition/ComplexTypes.json [hunk 3]
```diff
@@ -323,17 +324,19 @@
...
+    "FieldType": "Text",
     "ElementLabel": "Mobile number",
+    "RegularExpression": "^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[\\s0-9]{0,16}$",
     "HintText": "For example, 07665 545327",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "Applicant",
     "ListElementCode": "telephone",
```
Comment (geek-soft) [inline ccd-definition/ComplexTypes.json]: What about `landlinePhone`?

File: ccd-definition/ComplexTypes.json [hunk 4]
```diff
@@ -367,17 +370,19 @@
     "LiveFrom": "01/01/2017",
     "ID": "Solicitor",
     "ListElementCode": "mobile",
```
Comment (geek-soft) [inline ccd-definition/ComplexTypes.json]: What about `mobilePhone`?

File: ccd-definition/ComplexTypes.json [hunk 5]
```diff
@@ -367,17 +370,19 @@
...
+    "FieldType": "Text",
     "ElementLabel": "Solicitor's mobile number",
+    "RegularExpression": "^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[\\s0-9]{0,16}$",
     "HintText": "For example, 07665 545327",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "Solicitor",
     "ListElementCode": "telephone",
```
Comment (geek-soft) [inline ccd-definition/ComplexTypes.json]: What about `landlinePhone`?

File: ccd-definition/ComplexTypes.json [hunk 6]
```diff
@@ -1160,8 +1166,9 @@
     "LiveFrom": "01/01/2017",
     "ID": "RespondentType",
     "ListElementCode": "telephone",
-    "FieldType": "PhoneUK",
+    "FieldType": "Text",
```
Comment (geek-soft) [inline ccd-definition/ComplexTypes.json]: What about `landlinePhone`?

**Pull request merged**